src/objective-c/tests/InteropTests/InteropTests.m

```diff
@@ -40,6 +40,9 @@
 #define TEST_TIMEOUT 32
 #define STREAMING_CALL_TEST_TIMEOUT 64
 
+#define SMALL_PAYLOAD_SIZE 10
+#define LARGE_PAYLOAD_SIZE 314159
```

Comment:
Actually use this? :D

Reply:
good pt. Did a swipe through the rest of the tests and replacing the hard-coded size numbers with these.